src/Cli/dotnet/Program.cs

```diff
@@ -64,11 +64,6 @@ public static int Main(string[] args)
                 {
                     return ProcessArgs(args, startupTime);
                 }
-                catch (HelpException e)
```

Comment:
this clause, and the type caught in it, I removed because the HelpException type isn't used anywhere.